Utils/hdinsight-node-common/src/com/microsoft/azure/sqlbigdata/sdk/cluster/SqlBigDataLivyLinkClusterDetail.java

```diff
@@ -109,4 +106,11 @@ public SparkSubmitStorageType getDefaultStorageType() {
     public SparkSubmitStorageTypeOptionsForCluster getStorageOptionsType() {
        return SparkSubmitStorageTypeOptionsForCluster.BigDataClusterWithWebHdfs;
     }
+
+    @Override
+    @NotNull
+    public String mapInternalUrlToPublic(@NotNull String url) {
+        // TODO: we can extract application ID from internal URL and map it to output
```

Comment:
And fire an issue for this
```suggestion
        // FIXME: Hardcode for only output Yarn NM connection URL
        // we should extract application ID from internal URL and map it to output
```